src/components/utils/useActionHandlers.ts

```diff
@@ -10,16 +12,22 @@ interface UseActionHandlersResult<T> {
 /**
  * Emulates behaviour of system controls, that respond to Enter and Spacebar
  * @param callback
+ * @param keyCodes
  * @return {onKeyDown}
  */
-export function useActionHandlers<T>(callback?: AnyFunction): UseActionHandlersResult<T> {
+export function useActionHandlers<T>(
+    callback?: AnyFunction,
+    keyCodes: EKeyCode[] = [EKeyCode.ENTER, EKeyCode.SPACEBAR, EKeyCode.SPACEBAR_OLD],
```

Comment:
This hook is only for handling "action" (alternative for click on keyboard), therefore we should handle only Enter and Spacebar. No need in such option

Reply:
The reason why I added `keyCodes` arg – <a> tag reacts to the Enter by default and browser triggers click event. Thus, one should call this hook with `EKeyCode.SPACEBAR`, `EKeyCode.SPACEBAR_OLD`. For all other elements (except for <button>) one should `useActionHandlers` for Spacebar and Enter

Reply:
And I need this function in another project with `<a>` tags)

Reply:
Maybe one should restrict this parameter only for `EKeyCode.ENTER`, `EKeyCode.SPACEBAR` and `EKeyCode.SPACEBAR_OLD`?

Reply:
Why do you need this hook for `<a>` tags in the first place? Seems incorrect usage. The only purpose of this hook is to add keyboard functionality of the button to another element.

Reply:
To make `<a>` tags behave like buttons. But fine, really looks like incorrect usage as browsers consider Spacebar as page down action.

Reply:
Reverted the changes for the hook